src/commands/list_drive.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+import { arDriveFactory, cliArweave, cliWalletDao } from '..';
+import { ArDriveAnonymous } from '../ardrive';
+import { ArFSDAOAnonymous, ArFSPrivateFileOrFolderWithPaths, ArFSPublicFileOrFolderWithPaths } from '../arfsdao';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { DriveIdParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
+import { alphabeticalOrder } from '../utils/sort_functions';
+
+new CLICommand({
+	name: 'list-drive',
+	parameters: [DriveIdParameter, SeedPhraseParameter, WalletFileParameter],
```

Comment:
You're missing the drive password and drive key parameters for private drives. This doesn't work for private drives.

Reply:
c4e2a90